orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/OortHelper.groovy

```diff
@@ -58,7 +58,9 @@ class OortHelper {
                                                    String serverGroupName,
                                                    String location,
                                                    String cloudProvider) {
-    return new TargetServerGroup(convert(oortService.getServerGroup(account, location, serverGroupName) , Map))
+    return convertedResponse(Map) {
```

Comment:
This method actually expects an `Optional` to be returned ... it used to be this way but Groovy somehow allowed the `return new TargetServerGroup()`.